include/riakc.hrl

```diff
@@ -68,6 +68,9 @@
                       head | deletedvclock |
                       {n_val, pos_integer()} |
                       {sloppy_quorum, boolean()}.
+-type preflist_item() :: {parition, non_neg_integer()} | {node, binary()} |
+                         {primary, boolean()}.
+-type preflist() :: [preflist_item()].
```

Comment:
So, the preflist entries come back as a flat list? This type spec is confusing and possibly error-prone.